### DISCUSSION TITLE
Support for more modern Java versions

### DISCUSSION THREAD

**Discussion by mks-m** [apache/gluten#6032] [General]
Hello,

Is there a plan to support more modern versions and what is it?

**Comment by zhouyuan** [comment]
Hi,
Gluten should support JDK17 here, but only enabled on Spark 3.3 build to save CI resource. Gluten Spark 3.4/3.5 should work with JDK17 also.
https://github.com/apache/incubator-gluten/blob/main/.github/workflows/velox_docker.yml#L91
